man/git-reauthor.md

```diff
@@ -7,25 +7,26 @@ git-reauthor(1) -- Rewrite history to change author's identity
 
 ## DESCRIPTION
 
-Rewrite history to change author's identity.
+Lets you replace the author and/or committer identities in commits and tags.
 
-Goes through all existing commits and tags to selectively modify the identities:  
-- modify name and/or email  
-- affect author and/or committer identities  
-- specify the new name/email as parameters or use the ones from the Git config  
-- change all objects or only replace those from a specific identity (defined by its email)
+The command goes through all existing commits and tags in the history to selectively modify the identities present in those objects. All the other informations are preserved.  
```

Comment:
If it does this in all branches, then: `the history` -> `all branches`,
else if it's just the current branch, then: `the history` -> `the current branch`. (sorry, I don't remember how it worked, but you get the point.

> ~~All the other informations are preserved~~Commits not matching the query are left untouched.